src/binary/var_int.rs

```diff
@@ -1,6 +1,7 @@
 use crate::data_source::IonDataSource;
 use crate::result::{decoding_error, IonResult};
 use std::mem;
```

Comment:
In this file:
* Adding write support for [the low-level encoding primitive `VarInt`](https://amzn.github.io/ion-docs/docs/binary.html#varuint-and-varint-fields).